tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -373,6 +373,19 @@ increment_j_blue_estimator (const rpacket_t * packet, storage_model_t * storage,
     }
 }
 
+void
+increment_Edotlu_estimator (const rpacket_t * packet, storage_model_t * storage, int64_t line_idx)
+{
+  if (storage->line_lists_Edotlu != NULL)
+    {
+#ifdef WITHOPENMP
+#pragma omp atomic
+#endif
+    storage->line_lists_Edotlu[line_idx] += rpacket_get_energy (packet);
```

Comment:
Yes it depends on the transition, I just meant to say that the tau for each line is added to the estimator(s, the code above is also for each transition and each shell) once all monte carlo steps have finished.